edge-modules/MessagesAnalyzer/Settings.cs

```diff
@@ -77,12 +77,12 @@ class Settings
 
         public double ToleranceInMilliseconds { get; }
 
-        public bool LogAnalyticEnabled { get; }
+        public bool LogAnalyticsEnabled { get; }
 
-        public string LogAnalyticWorkspaceId { get; }
+        public string LogAnalyticsWorkspaceId { get; }
 
-        public string LogAnalyticSharedKey { get; }
+        public string LogAnalyticsSharedKey { get; }
```

Comment:
```suggestion
        public string LogAnalyticsSecret { get; }
```

Reply:
@and-rewsmith please change the `LogAnalyticsSecret` back to `LogAnalyticsSharedKey`. It is actually is a shared key from the Log Analytics.  Go to Log Analytics > Advance Settings > Linux/Windows > Primary shared key.

Reply:
I think this is a better name so I will change the "secret" name in other module